WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewController.swift

```diff
@@ -133,6 +145,18 @@ final class BulkUpdateViewController: UIViewController {
     @objc private func cancelButtonTapped() {
         viewModel.handleTapCancel()
     }
+
+    /// Displays the error `Notice`.
+    ///
+    private func displaySyncingError() {
+        let title = Localization.noticeUnableToSyncVariations
+        let actionTitle = Localization.noticeRetryAction
+        let notice = Notice(title: title, feedbackType: .error, actionTitle: actionTitle) { [weak self] in
+            self?.viewModel.activate()
+        }
```

Comment:
While this works right now I think this could be a little bit misleading in the feature, if/when we update the view model to perform more tasks. 

"Activate" to me sounds like a method that should be called only once.

I'm thinking that we should rename the method to `syncVariations` or provide a different one for only retrying.

What do you think?

Reply:
> "Activate" to me sounds like a method that should be called only once.

You are right, if in the future we have different functionality for retrying I think we can add then a separate method, for now I think `syncVariations`  is a more descriptive well suited name, so I renamed it!